portal-api/src/v1/routes.js

```diff
@@ -1,5 +1,4 @@
 const express = require('express');
```

Comment:
I've rearranged this file so that the requests are grouped by what router they are using, and put them in the order that they are run from `createApp.js`